frontends/web/src/routes/device/bitbox02/passphrase.tsx

```diff
@@ -251,11 +280,15 @@ class Passphrase extends Component<Props, State> {
                                     <SimpleMarkup key="tip-2" tagName="li" markup={t('passphrase.successEnabled.tipsList.1')} />
                                 </ul>
                             )}
-                        </WaitDialog>
-                    )}
-                    </div>
-                </div>
-            </div>
+                            <SimpleMarkup tagName="p" markup={t('passphrase.successEnabled.messageEnd')} />
+
+                        </ViewContent>
+                    </View>
+                )}
+                {/* <Guide>
+                    <Entry key="whatisapassphrase" entry={t('guide.passphrase.whatisapassphrase')} />
+                </Guide> */}
```

Comment:
is the guide entry not ready yet?

Reply:
Currently there is no additional guide for passphrase, but we now make the user read through the information instead of providing an optional guide.

Is there something missing that you'd like to see in the guide?

If yes then depending on what kind of information it is we should consider adding it to the information shown. Maybe the guide could contain some technical details that we don't want to show to regular users?